content/blog/2021-03-31-march-21-community-gems.md

```diff
@@ -0,0 +1,102 @@
+---
+title: March '21 Community Gems
+date: 2021-03-31
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-03-31/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/march-2021-community-gems/708
+tags:
+  - Discord
+  - Gems
+  - Pipelines
+  - Remote Storage
+---
+
+## DVC questions
+
+### [Q: Will DVC work with <my remote cloud storage of choice?>](https://discord.com/channels/485586884165107732/563406153334128681/821493606770409493)
+
+We recently had questions about this, specifically regarding Huawei Cloud and
+Backblaze B2 Storage. The answer is any cloud storage that has an s3 interface
+will work with DVC and both of the aforementioned do!  
+[Learn more about remote storage integrations here.](https://dvc.org/doc/command-reference/remote)
+
+Thanks to luke and Samuel H from Discord for asking these questions that led to
+this Gem! 💎
+
+### [Q: I had understood previously that DVC was not suitable for hyperparameter tuning. Has that changed?](https://discord.com/channels/485586884165107732/485596304961962003/820722752709328967)
+
+Yes indeed! With DVC 2.0, the capabilities have evolved quite a bit! We have
+introduced experiments and metrics which enables you to track and compare the
+different runs of your models with various hyperparameters. You can check out
+the documents [here](https://dvc.org/doc/start/experiments) and
+[here](https://dvc.org/doc/start/metrics-parameters-plots) to see all the
+details.
+
+Thanks to saif3r for helping us highlight the new features in DVC!
+
+### [Q: Is it possible to set up a DVC repo with pipelines which has all the data (cache, input, output) on another (local) location outside the repo?](https://discord.com/channels/485586884165107732/485596304961962003/819509440217874473)
```

Comment:
In the answer here it's good to mention that setup is also similar to this doc https://dvc.org/doc/use-cases/shared-development-server (it describes how to ` to keep your DVC cache on your mount`)